tests/baselines/reference/jsxAndTypeAssertion.js

```diff
@@ -28,18 +28,21 @@ var foo = /** @class */ (function () {
     return foo;
 }());
 var x;
-x = <any> {test} <any></any> };
+x = <any> {test}: <any></any> };
```

Comment:
The TypeScript generating this was `x = <any> { test: <any></any> };`. Garbage in, garbage out?